README.md

```diff
@@ -35,7 +55,8 @@ See below for a description of each method. The above snippet, assumes that the
 
 ## Example
 
-[See this example](http://bl.ocks.org/emepyc/ce259dd519f6a60d35d04c78b40ec425).
+We provide the following example to demonstrate how to embed an iHam session [See this example](http://bl.ocks.org/emepyc/ce259dd519f6a60d35d04c78b40ec425).
```

Comment:
I don't particularly like the wording in `embed an iHam session`. What about something like: `See [this example]() in this repository demonstrating how to use this widget in a custom web page` (or something similar)